### PR TITLE
Switch to s3+cloudfront

### PR DESCRIPTION
**Pull request by jgmize**
for https://github.com/mozmeao/infra/issues/312

### CODE REVIEW

File: Jenkinsfile [hunk 1]
```diff
@@ -1,46 +1,37 @@
...
-      if(fileExists("./Jenkinsfiles/${branch}.groovy")) {
-          println "Loading ./Jenkinsfiles/${branch}.groovy"
-          load "./Jenkinsfiles/${branch}.groovy"
-        } else {
-          println "Loading ./Jenkinsfiles/default.groovy"
-          load "./Jenkinsfiles/default.groovy"
+def syncS3(String bucket) {
+    stage ('s3 sync') {
+        try {
+          sh "cd release && aws s3 sync . s3://" + bucket +" --acl public-read --delete --profile irl --exclude 'docs/*'"
```
Comment (metadave) [inline Jenkinsfile]: the profile is incorrect here.

Comment (jgmize) [inline Jenkinsfile]: good catch, thanks! fixing now.

**Comment by jgmize** [comment]
@metadave I pushed https://github.com/mdn/viewsourceconf/pull/196/commits/a64aa36fee2044668cda3d4a52019ad9d30f2d67 to fix the AWS profile.

**Comment by metadave** [comment]
👍

**Pull request merged**